app/controllers/local_petitions_controller.rb

```diff
@@ -0,0 +1,30 @@
+require 'postcode_sanitizer'
+
+class LocalPetitionsController < ApplicationController
+  respond_to :html
+
+  def index
```

Comment:
That's an interesting idea, but we don't have a name -> id lookup for constituencies.  I guess it could be ``/petitions/local/westminister-north-1234/`` or something, but that leads us down the path of having urls where only part of the key is important and people can post urls like ``/petitions/local/look-at-what-these-idiots-care-about-1234/``  which seems bad.